spacy/pipeline/edit_tree_lemmatizer.py

```diff
@@ -115,29 +117,65 @@ def __init__(
 
         self.cfg: Dict[str, Any] = {"labels": []}
         self.scorer = scorer
+        self.lowercasing = model.has_ref("lowercasing_output")
 
     def get_loss(
         self, examples: Iterable[Example], scores: List[Floats2d]
     ) -> Tuple[float, List[Floats2d]]:
         validate_examples(examples, "EditTreeLemmatizer.get_loss")
-        loss_func = SequenceCategoricalCrossentropy(normalize=False, missing_value=-1)
-
-        truths = []
+        tree_loss_func = SequenceCategoricalCrossentropy(
+            normalize=False, missing_value=-1
+        )
+        lowercasing_loss_func = L2Distance(normalize=False)
```

Comment:
Since the layer is the logistic function (sigmoid), I'd expect a binary cross entropy loss. Due to how we handle gradients, I think L2Distance will have the same effect in this case, but it's not nice semantically and it will report incorrect losses.

Reply:
As we discussed offline, you plan to add a new loss function to Thinc to enable this.